### PR TITLE
[README.md] initial

### PR DESCRIPTION
**Pull request by szlldm**
new readme page.
patched together from pieces from here and there...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+```
+bash createDebFromSourceUbuntu.bash --deps-only
+```
+> Fedora:
+```
+bash createRpmFromSourceFedora.bash --deps-only
+```
+Build Avidemux:
+```
+bash bootStrap.bash --with-system-libass
```
Comment (eumagga0x2a) [inline README.md:34]: On Fedora, I always add also `--with-system-libmp4v2 --with-system-liba52 --with-system-libmad`. For Fedora, Avidemux is packaged by RPM Fusion which allows build dependencies to be installed by a straightforward

`sudo dnf builddep avidemux`

RPM Fusion needs to be enabled anyway to produce a reasonably feature-complete Avidemux build.

**Comment by eumagga0x2a** [comment]
I'm going to merge this as-is, further improvements can be added later.

File: README.md (hunk 1 above)
Comment (szlldm) [reply README.md:34]: Can you extend it? Im not familiar with Fedora :-)

Comment (eumagga0x2a) [reply README.md:34]: I'll do it some day :-)

**Pull request merged**